src/main/java/duke/message/Message.java

```diff
@@ -0,0 +1,158 @@
+package duke.message;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import duke.task.Task;
+import duke.task.TaskList;
+import javafx.geometry.Pos;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.HBox;
+
+/**
+ * Represent a message that can be displayed in the GUI.
+ */
+public class Message extends HBox {
+    private static final Image USER_IMAGE = new Image(
+            Message.class.getClassLoader().getResourceAsStream("/images/User.png"));
+    private static final Image BOT_IMAGE = new Image(
+            Message.class.getClassLoader().getResourceAsStream("/images/Bot.png"));
+    private final String content;
+
+    private final Label label;
+    private ImageView image;
+    /**
+     * Constructs a message with the given content.
+     *
+     * @param content The content of the message.
+     */
+    public Message(String content) {
+        this.content = content;
+        //Set label contents and formatting
```

Comment:
Perhaps there could be an empty space between `//` and `Set` to make it clearer? I notice the same issue with the following lines.
```suggestion
        // Set label contents and formatting
```